spatialscaper/core.py

```diff
@@ -587,22 +593,6 @@ def get_room_irs_wav_xyz(self, wav=True, pos=True):
             ir_sr = self.sr
         return all_irs, ir_sr, all_ir_xyzs
 
-    def get_format_irs(self, all_irs, fmt="mic"):
```

Comment:
Removed this function since we are now dealing with 4 channels for metu as well.

Reply:
Good move. Thanks